ocs_ci/ocs/resources/pod.py

```diff
@@ -1654,3 +1654,33 @@ def get_osd_removal_pod_name(osd_id, timeout=60):
     except TimeoutExpiredError:
         logger.warning(f"Failed to get pod ocs-osd-removal-{osd_id}")
         return None
+
+
+def check_toleration_on_pods(toleration_key=None):
+    """
+    Function to check toleration on pods
+
+    Args:
+        toleration_key (str): The toleration key to check
+
+    """
+    toleration_key = toleration_key if toleration_key else constants.TOLERATION_KEY
```

Comment:
consider setting the `toleration_key` function arg with a default value of `constants.TOLERATION_KEY`.
Then you will not need this line